Generate.py

```diff
@@ -8,9 +8,11 @@
 import urllib.parse
 import urllib.request
 from collections import Counter, ChainMap
+from itertools import chain
 from typing import Dict, Tuple, Callable, Any, Union
 
 import ModuleUpdate
+import worlds.AutoWorld
```

Comment:
running this import before the update() call will crash with missing dependencies

Reply:
ah PyCharm put it there automatically and since I didn't need to update didn't hit that. good to know.